prov/verbs/src/verbs_eq.c

```diff
@@ -164,6 +165,9 @@ fi_ibv_eq_cm_process_event(struct fi_ibv_eq *eq, struct rdma_cm_event *cma_event
 	case RDMA_CM_EVENT_ESTABLISHED:
 		*event = FI_CONNECTED;
 		entry->info = NULL;
+		ret = fi_ibv_modify_qp(cma_event->id->qp);
```

Comment:
> Even the unexpected queue requires posting receives to the QP.

Yes but this change fixes the issue where the sender could overrun receive queue in the QP. I think that might be rare when there is an unexpected queue. Also, I'm not sure if verbs/RDM protocol has any software flow control to handle that.

Nevertheless, this change can be added to verbs/RDM as well to avoid any potential queue overruns.